### PR TITLE
MHR search report UXA updates.

### PR DESCRIPTION
**Pull request by doug-lovett**
Signed-off-by: Doug Lovett <doug@diamante.ca>

*Issue #:* /bcgov/entity#12177

*Description of changes:*
- MHR search report updates from UXA review.
- Revert to sharing staff a single staff role between MHR and PPR (search and registration history single account ID).
- Update to PPR db function for generating a vehicle search key: remove non-alphanumeric characters.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by dimak1 — APPROVED** [review]
LGTM!

File: mhr_api/report-templates/template-parts/search-result-ppr/amendmentStatement.html [hunk 1]
```diff
@@ -58,8 +58,6 @@
 </div>
 {% endif %}
 
-<div class="container pt-4">
-  [[search-result-ppr/registeringParty.html]]
+<div class="container pt-4 pb-4">
+    [[search-result-ppr/registeringParty.html]]
```
Comment (cameron-eyds) [inline mhr_api/report-templates/template-parts/search-result-ppr/amendmentStatement.html:62]: Intentional extra spacing?

**Review by cameron-eyds — APPROVED** [review]
Looks good, just a very minor question!

File: mhr_api/report-templates/template-parts/search-result-ppr/amendmentStatement.html (hunk 1 above)
Comment (doug-lovett) [reply mhr_api/report-templates/template-parts/search-result-ppr/amendmentStatement.html:62]: Yes, the UXA feedback was to remove the separator line but retain the spacing.

Comment (cameron-eyds) [reply mhr_api/report-templates/template-parts/search-result-ppr/amendmentStatement.html:62]: kk right on!

**Review by dimak1 — APPROVED** [review]

**Pull request merged**